cypress/fixtures/insights-cleanup.sh

```
@@ -0,0 +1,80 @@
+rawHost="${HOST:-localhost:8080}"
+if [[ "$rawHost" == *"/hub"* ]]; then
+  host="$rawHost"
+else
+  host="${rawHost}/hub"
+fi
+
+if [[ ! "$host" =~ ^https?:// ]]; then
+  host="https://${host}"
+fi
+
+export TOKEN=$(curl -kSs -d "{\"user\":\"${USERNAME}\",\"password\":\"${PASSWORD}\"}" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)